ml-core-base/build.gradle

```diff
@@ -0,0 +1,15 @@
+apply from: configs.androidLibrary
+
+dependencies {
+    implementation "androidx.annotation:annotation:$androidxAnnotationVersion"
+    ktlint "com.pinterest:ktlint:$ktlintVersion"
```

Comment:
Probably shouldn't be done in this PR, but we should add this to the android library gradle file.

Reply:
Tracked [here](https://jira.corp.stripe.com/browse/MOBILESDK-953)